tests/TestRunner/tmp_cluster.py

```diff
@@ -63,12 +63,12 @@ def __enter__(self):
     def __exit__(self, xc_type, exc_value, traceback):
         super().__exit__(xc_type, exc_value, traceback)
         if self.remove_at_exit:
-            shutil.rmtree(self.tmp_dir)
+            shutil.rmtree(self.tmp_dir, ignore_errors=True)
 
     def close(self):
         super().__exit__(None, None, None)
         if self.remove_at_exit:
-            shutil.rmtree(self.tmp_dir)
+            shutil.rmtree(self.tmp_dir, ignore_errors=True)
```

Comment:
Why are we ignoring the FileNotFound error rather than fixing it? With that error, will we still satisfy the contract of `remove_at_exit`? And if yes, won't `ignore_error=True` ignore _all_ errors including ones that can potentially break the contract `remove_at_exit`?